### PR TITLE
copy to clipboard prototype

### PR DESCRIPTION
**Pull request by jbarnicle**
Rudimentary prototype.

### CODE REVIEW

File: atf_eregs/templates/atf_eregs/poster_wrapper.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
     </p>
   </div>
 
+  <a href="#" class="clipboard-link">Copy to clipboard</a>
   <div class="poster extract">
```
Comment (cmc333333) [inline atf_eregs/templates/atf_eregs/poster_wrapper.html]: Let's go w/ progressive enhancement over including extra markup. To me that means adding a "clipboard-link" to the `div` and inserting the link via JS

Comment (jbarnicle) [inline atf_eregs/templates/atf_eregs/poster_wrapper.html]: Thought about that - works for me.

**Comment by jbarnicle** [comment]
Reopening to add custom css

**Comment by jbarnicle** [comment]
After thinking about it some more, it feels like this particular "copy to clipboard" styling is very particular to the poster content - which is being styled in this branch.  So I though that the css for this should also go in this branch.  Thoughts?

**Comment by cmc333333** [comment]
Good point! It might also be the case that the functionality (JS) should also belong in the ATF-specific land. Given that we don't have a working infrastructure for that at the moment, though, I agree with this split strategy.

Looks like you'll want to add a new class/attribute for https://github.com/18F/regulations-site/pull/75#discussion_r50362396, too?

**Comment by cmc333333** [comment]
:+1:

**Pull request merged**